README.md

```diff
@@ -85,13 +73,17 @@ cordova.plugins.instabug.activate(
 
 You can change the invocation event with any of the following: `'button'`, `'screenshot'`, `'swipe'`, or `'shake'`.
 
-##### Ionic
+⚠️ TypeScript users, make sure you declare cordova at the beginning of your App TS class:
 
-If you're using this plugin with Ionic, you need to add this line at the beginning of your **app.component.ts** class.
```

Comment:
Do you think removing this is necessary? I thought using the exact file name was useful.

Reply:
The file doesn't necessarily have to be named like that. Also, our integration doesn't have to be placed in that file.
Actually, this tip is general knowledge for TS users, if they're using Cordova, they most probably already know it (we can actually remove the whole tip :D)